models/index.js

```diff
@@ -19,3 +20,11 @@ exports.fetchReviews = () => {
     return reviews;
   });
 };
+
+exports.fetchReviewById = (id) => {
+  const queryStr = format(`SELECT * FROM reviews WHERE review_id = %L;`, id);
```

Comment:
an interesting place to use a %L here, but i would say that is not needed for this query and this can be done in a simpler way.